pct.sh

```diff
@@ -103,26 +103,39 @@ java \
 	-skipTestCache true
 
 if grep -q -F -e '<status>INTERNAL_ERROR</status>' pct-report.xml; then
-	echo PCT failed
+	echo 'PCT failed with internal error' >&2
+	cat pct-report.xml
+	exit 1
+elif grep -q -F -e '<status>COMPILATION_ERROR</status>' pct-report.xml; then
+	echo 'PCT failed with compilation error' >&2
 	cat pct-report.xml
 	exit 1
```

Comment:
Handling another case in the enum: https://github.com/jenkinsci/plugin-compat-tester/blob/0e40e65a81e1f61aad64c2817f5cf16b1dd898d5/plugins-compat-tester-model/src/main/java/org/jenkins/tools/test/model/TestStatus.java#L34